src/Modules/PaymentModule/components/CheckoutForm/CheckoutForm.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Button } from "@mui/material"
+import { CardElement, useElements, useStripe } from "@stripe/react-stripe-js"
+import axios from "axios";
+import { FormEvent } from "react";
+
+
+export const CheckoutForm =  () => {
+
+  const elements = useElements();
+  const stripe = useStripe();
+
+  const submitHandler = async (e:FormEvent)=>{
+
+    e.preventDefault();
+
+    if (!elements || !stripe) return;
+    const cardElement = elements.getElement(CardElement);
+    
+    if (!cardElement) return;
+    const {error,token} = await stripe.createToken(cardElement);
+
+    if (error){
+      console.log({tokenError:error});
+      return;
+    }
+    console.log({token});
+
+    try{
+      const response = await axios.post('https://upskilling-egypt.com:3000/api/v0/portal/booking/66666628a17944edfda3da0a/pay',
+        {
+          token:token.id
+        },
+        {
+          headers:{Authorization:'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NWExNGIxYTI4M2I1NmY1NjgyMTMyNGYiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcxODAyNTcyNywiZXhwIjoxNzE5MjM1MzI3fQ.TI2XaB5g8PRNqwIV_ZV2RuQ90qwGZrOvWhcCrH1A9_E'}
+        }
+      )
+
+    } catch(error){
+      console.log({backendError: error});
+      
+    }
+
+    
+  }
+
+  return (
+    <>
+
+    <Box 
```

Comment:
We can render this Box as a form by giving it the prop component and then typing "form"
`component="form"`